readme.md

```diff
@@ -1,4 +1,4 @@
-# pkg-upgrader 
+# pkg-upgrader
 
 <img src="screenshot.gif" width="440" align="right">
```

Comment:
is it worth re-doing the animated gif here to show cli improvements?

Reply:
👍

Reply:
Do we want a gif of nui-widgets-codemods on public github or will that just be more confusing?

Reply:
Definitely no.  Just a generic package

Reply:
Definitely no.  Just a generic package

Reply:
Adding an issue to do that :)